README.md

```diff
@@ -541,6 +541,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Chamberlain Ezigbo](https://github.com/Chamberezigbo)
   - [Chandraprakash Darji](https://github.com/Chandraprakash-Darji)
   - [Chandula Janith](https://github.com/RedEdge967)
+  - [Chandan Gupta](https://github.com/chandanpc)
```

Comment:
Please add your name in line 542, after Chamberlain Ezigbo.
Let us know if you need any help doing that! 🙂